src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -26,7 +26,7 @@ export const initialArtworkFilterState = {
 export interface ArtworkFilters {
   acquireable?: boolean
   artist_id?: string
-  at_auction?: boolean
+  atAuction?: boolean
```

Comment:
Some of the other fields here (`inquireable_only`, `for_sale`) need to change similarly (and below occurrences too).